tests/02_eradiate/01_unit/test_frame.py

```diff
@@ -82,7 +85,7 @@ def test_transform_azimuth_from_east_right(to_convention, initial, normalize):
         to_convention=to_convention,
         normalize=normalize,
     )
-    assert np.allclose(result, expected)
+    assert np.allclose(result, expected), np.rad2deg(result)
```

Comment:
What is happening here?

Reply:
It's just a thing I do to make debugging simpler: the second argument of the assert statement gets printed upon failure, and it's sometimes much more comprehensive than the traceback. If the test succeeds, you see nothing. In that case, I just output the result vector converted to degrees to see better what was going on. I'll clean that up.